src/ExportButton.tsx

```diff
@@ -181,7 +184,12 @@ const getTopNVariantsWithinLimit = (
     }
   }
 
-  return [...previousVariants, hadTrimmedVariants];
+  if (hadTrimmedVariants) {
+    // If we had to trim variants, then we want the component to be classified as complex
+    return [previousVariants[0], true, hadTrimmedVariants];
+  } else {
+    return [...previousVariants, hadTrimmedVariants];
+  }
```

Comment:
The latest spec specifies that trimmed components should be classified as complex :)